### ISSUE TITLE
Syntax error when changing default length

### ISSUE THREAD

**Issue by eagle-r** [deanishe/alfred-pwgen#5]
The workflow is crashing whenever I try to change the default length in the options. The debug output below seems to show a syntax error of some sort.

```[2016-11-30 10:08:59][STDERR: input.scriptfilter] 10:08:59 workflow.py:2140 DEBUG    Workflow version : 1.3.0
10:08:59 workflow.py:1543 DEBUG    Reading settings from `/Users/richard/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred-pwgen/settings.json` ...
10:08:59 workflow.py:2401 DEBUG    Update check not due
10:08:59 pwgen.py:171 DEBUG    Set log level to DEBUG
10:08:59 pwgen.py:173 DEBUG    args : {u'--debug': False,
 u'--help': False,
 u'--length': False,
 u'--quiet': False,
 u'--verbose': False,
 u'--version': False,
 u'<genid>': None,
 u'<key>': None,
 u'<length>': None,
 u'<query>': u'',
 u'<strength>': None,
 u'<value>': None,
 u'conf': True,
 u'generate': False,
 u'set': False,
 u'toggle': False}
10:08:59 workflow.py:1778 DEBUG    Loading cached data from : /Users/richard/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/net.deanishe.alfred-pwgen/__workflow_update_status.cpickle
10:08:59 workflow.py:2349 DEBUG    update_data : {u'available': False, u'version': u'v1.3', u'download_url': u'https://github.com/deanishe/alfred-pwgen/releases/download/v1.3/Password.Generator-1.3.alfredworkflow'}
10:08:59 __init__.py:219 DEBUG    Imported built-in generators from `generators.gen_basic`
10:08:59 __init__.py:219 DEBUG    Imported built-in generators from `generators.gen_dictionary`
10:08:59 __init__.py:219 DEBUG    Imported built-in generators from `generators.gen_german`
10:08:59 __init__.py:219 DEBUG    Imported built-in generators from `generators.gen_pronounceable`
10:08:59 __init__.py:219 DEBUG    Imported built-in generators from `generators.gen_pronounceable_markov`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `Dictionary`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `Pronounceable Markov chain`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `German Pronounceable Markov`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `Pronounceable Nonsense`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `ASCII`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `German`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `Alphanumeric`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `German Alphanumeric`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `Clear Alphanumeric`
10:08:59 __init__.py:263 DEBUG    Loaded generator : `Numeric`
10:08:59 workflow.py:2331 DEBUG    Set last run version : 1.3.0
10:08:59 workflow.py:2176 DEBUG    Workflow finished in 0.113 seconds.
[2016-11-30 10:09:06][ERROR: action.script] 10:09:06 workflow.py:2140 DEBUG    Workflow version : 1.3.0
10:09:06 workflow.py:1543 DEBUG    Reading settings from `/Users/richard/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred-pwgen/settings.json` ...
10:09:06 workflow.py:2401 DEBUG    Update check not due
10:09:06 pwgen.py:171 DEBUG    Set log level to DEBUG
10:09:06 pwgen.py:173 DEBUG    args : {u'--debug': False,
 u'--help': False,
 u'--length': False,
 u'--quiet': False,
 u'--verbose': False,
 u'--version': False,
 u'<genid>': None,
 u'<key>': u'pw_length',
 u'<length>': None,
 u'<query>': None,
 u'<strength>': None,
 u'<value>': None,
 u'conf': False,
 u'generate': False,
 u'set': True,
 u'toggle': False}
44:45: syntax error: Expected end of line but found “"”. (-2741)
10:09:06 workflow.py:2331 DEBUG    Set last run version : 1.3.0
10:09:06 workflow.py:2176 DEBUG    Workflow finished in 0.081 seconds.```

**Comment by skyzyx** [comment]
workflow.py seems to be set to log.DEBUG by default.

**Comment by deanishe** [comment]
Fixed in v2.0
